server/api/apps/shared_models/models/statistics_models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from .quiz_models import Question, Answer
+from django.contrib.auth.models import User
```

Comment:
https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#referencing-the-user-model